app/menus/AccountMenu.js

```diff
@@ -91,6 +43,65 @@ function AccountMenu(props: Props) {
     }
   }, [menu, ui.theme, previousTheme]);
 
+  const items = () =>
+    filterTemplateItems([
```

Comment:
Do we need to call `filterTemplateItems` here? It's the first thing that's called inside of `<Template>`

Reply:
No need.

This is also done in CollectionMenu. Will fix.